CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -153,57 +155,76 @@ class EditingViewController: UIViewController {
-            Interest(name: "Art", categories: "lorem, lorem, lorem, lorem, lorem", image: "art"),
-            Interest(name: "Business", categories: "lorem, lorem, lorem, lorem, lorem", image: "business"),
-            Interest(name: "Dance", categories: "lorem, lorem, lorem, lorem, lorem", image: "dance")
-        ]
-        let moreInterests: [Interest] = [
-            Interest(name: "Design", categories: "lorem, lorem, lorem, lorem, lorem", image: "design"),
-            Interest(name: "Fashion", categories: "lorem, lorem, lorem, lorem, lorem", image: "fashion"),
-            Interest(name: "Fitness", categories: "lorem, lorem, lorem, lorem, lorem", image: "fitness"),
-            Interest(name: "Food", categories: "lorem, lorem, lorem, lorem, lorem", image: "food"),
-            Interest(name: "Humanities", categories: "lorem, lorem, lorem, lorem, lorem", image: "humanities"),
-            Interest(name: "Music", categories: "lorem, lorem, lorem, lorem, lorem", image: "music"),
-            Interest(name: "Photography", categories: "lorem, lorem, lorem, lorem, lorem", image: "photography"),
-            Interest(name: "Reading", categories: "lorem, lorem, lorem, lorem, lorem", image: "reading"),
-            Interest(name: "Sustainability", categories: "lorem, lorem, lorem, lorem, lorem", image: "sustainability"),
-            Interest(name: "Technology", categories: "lorem, lorem, lorem, lorem, lorem", image: "tech"),
-            Interest(name: "Travel", categories: "lorem, lorem, lorem, lorem, lorem", image: "travel"),
-            Interest(name: "TV & Film", categories: "lorem, lorem, lorem, lorem, lorem", image: "tvfilm")
-        ]
-        let yourGroups: [Group] = [
-            Group(name: "Apple", image: ""),
-            Group(name: "banana", image: ""),
-            Group(name: "Cornell AppDev", image: "")
-        ]
-        let moreGroups: [Group] = [
-            Group(name: "dandelion", image: ""),
-            Group(name: "giraffe", image: ""),
-            Group(name: "heap", image: ""),
-            Group(name: "Igloo", image: ""),
-            Group(name: "Jeans", image: "")
-        ]
+        setupSections()
+        setupNavigationBar()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(false, animated: false)
+    }
 
-        let yourSection: Section
-        let moreSection: Section
+    private func setupSections() {
         if isShowingGroups {
-            yourSection = Section(type: .yours, items: yourGroups.map { .group($0) })
-            moreSection = Section(type: .more, items: moreGroups.map { .group($0) })
+            setSectionsFromGroups()
         } else {
-            yourSection = Section(type: .yours, items: yourInterests.map { .interest($0) })
-            moreSection = Section(type: .more, items: moreInterests.map { .interest($0) })
+            setSectionsFromInterests()
```

Comment:
very small nit but i feel like `setupSectionsFromGroups`/`setupSectionsFromInterests` would be slightly more consistent with the naming of the different "setup" functions